### PR TITLE
fix GROW-1341(Carousel tweaks for artist series and related collections)

### PR DESCRIPTION
**Pull request by jiahaodavid**
addresses: [GROW-1341](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=4&projectKey=GROW&modal=detail&selectedIssue=GROW-1341)

In this PR, we addresses the following:

1. No arrow when collections < 4 (desktop view)
Screen Shot 2019-06-26 at 1 44 42 PM
2. No infinite scroll,  just showing the remainder if next set is less than 4. 
    Also, reaching the end of the rail and starting from the beginning of the rail -> No arrow on the side.
    (desktop view)
Jun-26-2019 15-00-11

3. Can be swiped now if the artist only have 4 collections (mobile view)
Jun-26-2019 15-07-05

4. Swipe collection one by one. Not 4 collections each swipe. (mobile view)
Jun-26-2019 14-45-33

### CODE REVIEW

File: src/Apps/Artist/Components/ArtistCollectionsRail/__tests__/ArtistCollectionsRail.test.tsx [hunk 1]
```diff
@@ -45,6 +45,14 @@ describe("CollectionsRail", () => {
     expect(component.find(ArtistCollectionEntity).length).toBe(0)
   })
 
+  it("No arrows when there are less than 5 collections", () => {
+    const component = getWrapper()
+    // const { Component, dispatch } = mockTracking(RelatedCollectionsRail)
+    // const component = mount(<Component {...props} />)
```
Comment (xtina-starr) [inline src/Apps/Artist/Components/ArtistCollectionsRail/__tests__/ArtistCollectionsRail.test.tsx]: @jiahaodavid we will want to remove the these comments.

File: src/Components/RelatedCollectionsRail/__tests__/RelatedCollectionsRail.test.tsx [hunk 2]
```diff
@@ -45,6 +45,14 @@ describe("CollectionsRail", () => {
     expect(component.find(RelatedCollectionEntity).length).toBe(0)
   })
 
+  it("No arrows when there are less than 5 collections", () => {
+    const component = getWrapper()
+    // const { Component, dispatch } = mockTracking(RelatedCollectionsRail)
+    // const component = mount(<Component {...props} />)
```
Comment (xtina-starr) [inline src/Components/RelatedCollectionsRail/__tests__/RelatedCollectionsRail.test.tsx]: Same here.

Comment (jiahaodavid) [reply src/Components/RelatedCollectionsRail/__tests__/RelatedCollectionsRail.test.tsx]: Got it.

**Comment by xtina-starr** [comment]
@jiahaodavid looks like this has a merge conflict which is preventing it from getting merged. Let me know if you need help with fixing.

**Comment by artsyit** [comment]
:rocket: PR was released in v16.18.7 :rocket:

**Pull request merged**